From: Robert Luberda <robert@debian.org>
Date: Thu, 9 Apr 2015 23:34:55 +0200
Subject: 28 Fix gcc warning

Fix the following warning:
lex.c:128:2: warning: implicit declaration of function 'mkostemp' [-Wimplicit-function-declaration]
---
 lex.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/lex.c b/lex.c
index 8c2eae4..a91d220 100644
--- a/lex.c
+++ b/lex.c
@@ -30,12 +30,16 @@
  * SUCH DAMAGE.
  */
 
+#define _GNU_SOURCE // for mkostemp in stdlib.h
+#include <stdlib.h>
+
 #include "rcv.h"
 #include <errno.h>
 #include <fcntl.h>
 #include "extern.h"
 #include <bsd/err.h>
 
+
 /*
  * Mail -- a mail program
  *
